static/app/views/profiling/profilesProvider.tsx

```diff
@@ -144,12 +143,7 @@ export function TransactionProfileProvider({
         setProfile({type: 'resolved', data: p});
       })
       .catch(err => {
-        // XXX: our API client mock implementation does not mimick the real
-        // implementation, so we need to check for an empty object here. #sad
-        const isEmptyObject = err.toString() === '[object Object]';
-        const message = isEmptyObject
-          ? t('Error: Unable to load profiles')
-          : err.toString();
+        const message = err.toString();
```

Comment:
@JonasBa the comment indicated that the check as only there for testing, so I removed it because mocking now does return a `RequestError`.

But I’ve been thinking if `err.toString()` is the correct behaviour for production, as it will likely yield a string with `RequestError: ` as prefix. Would we not want the string `Error: Unable to load profiles` in production as well?

Reply:
Yes, iirc that is what is displayed in the UI. Lets leave a note and we can revisit this!